source/includes/properties/_00-list-show.md.erb.md

```diff
@@ -0,0 +1,19 @@
+## List & Show
+```shell
+$ curl http://goserver.com:8153/go/properties/PipelineName/1243/StageName/1/JobName 
```

Comment:
See the curl convention in other pages. Specifically, we quote everyting in the curl command using a single quote.